### PR TITLE
Add Talisman Eth API and Substrate API Wallet Pages

### PR DESCRIPTION
**Pull request by themacexpert**
### Description

Adds Talisman Pages for Eth API and Substrate API (different pages for each)

### Checklist

- [x] If this page requires a disclaimer, I have added one
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects

### CODE REVIEW

File: builders/interact/ethereum-api/wallets/talisman.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+1. Select the **Ethereum** account type
+2. Provide a name for your account
+3. Paste in your seed or private key
+4. If you imported a mnenomic seed phrase in the prior step, select which accounts you'd like to import 
+5. Press **Import**
+
+![Import existing account final steps](/images/builders/interact/ethereum-api/wallets/talisman/talisman-10.png)
+
+This guide focused specifically on configuring Talisman to work with your EVM ContainerChain, but Talisman is also a full featured wallet for Substrate (Polkadot) accounts. Under the Substrate API section you'll find a similar tutorial for configuring Talisman to work with Substrate-based chains.
```
Comment (eshaben) [inline builders/interact/ethereum-api/wallets/talisman.md:97]:
```suggestion
This guide focused specifically on configuring Talisman to work with your EVM ContainerChain, but Talisman is also a full-featured wallet for Substrate (Polkadot) accounts. Under the Substrate API section, you'll find a similar tutorial for configuring Talisman to work with Substrate-based chains.
```

File: builders/interact/ethereum-api/wallets/talisman.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+First, download and install the [Talisman extension](https://www.talisman.xyz/){target=_blank}. This guide will first cover creating a new wallet and later address importing an existing one. Review the terms and conditions then press **Get Started**. 
+
+![Get started with Talisman](/images/builders/interact/ethereum-api/wallets/talisman/talisman-1.png)
+
+On the following screen you'll be prompted to create a password to secure your new wallet. 
+
+![Enter password for Talisman Wallet](/images/builders/interact/ethereum-api/wallets/talisman/talisman-2.png)
+
+To create your first Ethereum account, take the following steps:
```
Comment (eshaben) [inline builders/interact/ethereum-api/wallets/talisman.md:26]: This should be its own section "Creating an Ethereum Account" and then the importing section can be after this one

**Review by eshaben — CHANGES_REQUESTED** [review]
Nice job!

The talisman light mode icons aren't appearing because they have `-light.png` at the end of the file name. So just need to remove that so they're both `talisman.png`.

Also, it doesn't look like they are the same size 256x256 as the wallets icon, can you please double check and fix as needed? Colors look like they are on point 👌

The setting up section (after splitting out the creating an account bit to a new section) can be a text snippet shared between the ethereum and substrate pages!

**Comment by themacexpert** [comment]
Thanks for the review @eshaben

File: builders/interact/substrate-api/wallets/talisman.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+2. Provide a name for your account
+3. Paste in your seed
+4. Select which accounts you'd like to import 
+5. Press **Import**
+
+![Import existing account final steps](/images/builders/interact/substrate-api/wallets/talisman/talisman-13.png)
+
+## Connecting Talisman to the Tanssi Dancebox TestNet {: #connecting-talisman-to-the-tanssi-dancebox-testnet }
+
+Talisman already comes pre-configured with support for the Tanssi Dancebox Testnet. You just need to first ensure that you have enabled support for TestNets on Talisman. Remember that the Tanssi Dancebox Testnet itself is not a ContainerChain; it is the Substrate-based parachain that orchestrates and manages the launch of Tanssi ContainerChains. To configure your Talisman to work with Dancebox, take the following steps:
```
Comment (eshaben) [inline builders/interact/substrate-api/wallets/talisman.md:65]:
```suggestion
Talisman already comes pre-configured with support for the Tanssi Dancebox Testnet. You just need to first ensure that you have enabled support for TestNets on Talisman. Remember that the Tanssi Dancebox TestNet itself is not a ContainerChain; it is the Substrate-based parachain that orchestrates and manages the launch of Tanssi ContainerChains. To configure your Talisman to work with Dancebox, take the following steps: 
```

**Review by eshaben — APPROVED** [review]
Looks good to me! I'll approve now, but I think after committing the one capitalization fix, I'll need to approve again haha... if that's the case, just tag me or request my review again please! 🙂

**Review by eshaben — APPROVED** [review]

**Pull request merged**